moveit_ros/planning/kinematics_plugin_loader/src/kinematics_parameters.yaml

```diff
@@ -4,23 +4,30 @@ kinematics:
     default_value: kdl_kinematics_plugin/KDLKinematicsPlugin,
     description: "Kinematics solver",
     validation: {
-      not_empty<>: []
+      not_empty<>: [],
+      one_of<>: [[kdl_kinematics_plugin/KDLKinematicsPlugin,
+                  prbt_manipulator/IKFastKinematicsPlugin,
+                  bio_ik/BioIKKinematicsPlugin,
+                  lma_kinematics_plugin/LMAKinematicsPlugin,
+                  srv_kinematics_plugin/SrvKinematicsPlugin,
+                  cached_ik_kinematics_plugin/CachedKDLKinematicsPlugin,
+                  cached_ik_kinematics_plugin/CachedSrvKinematicsPlugin ]]
     }
   }
   kinematics_solver_search_resolution: {
     type: double,
     default_value: 0.1,
     description: "kinematics solver search resolution",
     validation: {
-      lower_bounds<>: [ 0.0 ]
+      gt_eq<>: [ 0.0 ]
     }
   }
   kinematics_solver_timeout: {
     type: double,
     default_value: 0.05,
     description: "kinematics solver timeout",
     validation: {
-      lower_bounds<>: [ 0.0 ]
+      gt_eq<>: [ 0.0 ]
```

Comment:
What does a timeout of 0.0 mean? If that has a special meaning please add that to the description. Otherwise I think this should just be `gt`.

Reply:
Forgot to make this change. Thanks for catching!